schematypes.go

```diff
@@ -1901,3 +1901,126 @@ type DisassembledInstruction struct {
 
 // InvalidatedAreas: Logical areas that can be invalidated by the 'invalidated' event.
 type InvalidatedAreas string
+
+// Mapping of request commands and corresponding struct constructors that
+// can be passed to json.Unmarshal.
+var requestCtor = map[string]messageCtor{
+	"cancel":        func() Message { return &CancelRequest{} },
+	"runInTerminal": func() Message { return &RunInTerminalRequest{} },
+	"initialize": func() Message {
+		return &InitializeRequest{
+			Arguments: InitializeRequestArguments{
+				// Set the default values specified here: https://microsoft.github.io/debug-adapter-protocol/specification#Requests_Initialize.
+				LinesStartAt1:   true,
+				ColumnsStartAt1: true,
+				PathFormat:      "path",
+			}}
+	},
+	"configurationDone":         func() Message { return &ConfigurationDoneRequest{} },
```

Comment:
Not critical: but how is the spacing lined up here, but not above?

Reply:
seems like this is just how the go formatter works!